src/lib/init.ts

```diff
@@ -519,7 +720,78 @@ function getPackageJSON(isJS: boolean) {
     },
     devDependencies: {
       vite: "4.4.5",
-      ...(isJS ? {} : { typescript: "5.0.2" }),
+      ...(isJS
+        ? {}
+        : {
+            [TYPESCRIPT_NPM_DEPENDENCY]:
+              CLIPackageJSON.devDependencies[TYPESCRIPT_NPM_DEPENDENCY],
+          }),
+    },
+  };
+}
+
+function getGatewayPackageJSON(isJS: boolean) {
+  return {
+    private: true,
+    name: "gateway",
+    description: "Fluence gateway for running aqua function via HTTP methods",
+    version: "0.0.1",
+    type: "module",
+    keywords: [],
+    scripts: {
+      ...(isJS
+        ? {}
+        : {
+            build: "rm -rf ./dist && tsc -p tsconfig.json",
+          }),
+      start: isJS
+        ? "node --no-warnings src/server.js"
+        : "node --no-warnings --loader ts-node/esm src/server.ts",
+    },
+    dependencies: {
+      [JS_CLIENT_NPM_DEPENDENCY]:
+        CLIPackageJSON.dependencies[JS_CLIENT_NPM_DEPENDENCY],
+      fastify: "4.25.2",
+      "@fastify/rate-limit": "9.1.0",
+      "@sinclair/typebox": "0.32.11",
     },
+    ...(isJS
+      ? {}
+      : {
+          devDependencies: {
+            "@types/node": "20.11.5",
+            "@fastify/type-provider-typebox": "4.0.0",
+            [TYPESCRIPT_NPM_DEPENDENCY]:
+              CLIPackageJSON.devDependencies[TYPESCRIPT_NPM_DEPENDENCY],
+            "ts-node": "10.9.2",
+          },
+        }),
   };
 }
+
+const GATEWAY_README_CONTENT = `## Fluence HTTP Gateway
+
+Here you can find an example of simple Fluence HTTP gateway.
+The gateway allows you to map Aqua functions to http requests i.e., you can create HTTP route and handle the incoming request by executing some Aqua function.
+You can check it out and test this repo.
+
+### Start gateway
+
+- \`npm install\`
+- \`npm run start\`
+- \`curl -X POST http://localhost:8080/my/callback/hello -H "ACCESS_TOKEN: abcdefhi" -H 'Content-Type: application/json' -d '{"name": "Fluence" }'\`
```

Comment:
Let's also explain that there are `runDeployedServices` and `showSubnet` that correspond to `fluence run -f runDeployedServices` and `fluence run -f showSubnet`, and they work after `deal deploy`?